Test and Automate Features behind the Flags using Cypress Tests

arpendu11 opened the pull request:
Please review the changes you did in this PR and check these points to make sure that the code examples are of high quality:

- [ ] the source branch for this PR is NOT the master branch of your fork (please create a dedicated branch for the PR)
- [ ] the PR doesn't contain code that is irrelevant to the topic of the article
- [ ] the PR doesn't contain any other files that are irrelevant to the topic of the article
- [ ] the PR contains a README.md file that explains in a sentence or two what the code example is about and links to the article on reflectoring.io (if you added to an existing module, please add the link to the existing README.md file)
- [ ] the code is covered with tests where it makes sense
- [ ] if you have created a new Maven or Gradle module, please add it to the CI build (search for "# ADD NEW MODULES HERE" in `build-all.sh`)
- [ ] used the latest Gradle or Maven version
- [ ] used Java 11 or later 
- [ ] the [GitHub Actions build](https://github.com/thombergs/code-examples/actions?query=workflow%3ACI) is successful for your PR branch

**Feel free to raise the PR even if the checklist hasn't been worked through, yet**. But please only flag it to be reviewed after going through the checklist above.

Also feel free to reach out if you have any trouble with any of the steps.

thombergs reviewed (changes requested):
Can you please remove the node_modules folder from the PR?

arpendu11 commented:
> Can you please remove the node_modules folder from the PR?

I had added a folder from the other article here by mistake. I have removed that as we have a separate PR for that.
